scripts/game.js

```diff
@@ -0,0 +1,64 @@
+var rows = 50;
+var cols = 100;
+
+var playing = false;
+
+// define grid using 'new' operator to create an instance of the grid object
+var grid = new Array(rows);
+var nextGrid = new Array(rows);
+
+var timer;
+var reproductionTime = 100;
+
+
+
+function buildGrid() {
+  for (var i = 0; i < rows; i++) {
+    grid[i] = new Array(cols);
+    nextGrid[i] = new Array(cols);
+  }
+}
+function loadGrid() {
+    buildGrid();
+}
+
+// click handler for when user clicks cell on grid and turns cell "on" or "off"
+function clickHandler() {
```

Comment:
Hey! I think there's a missing curly bracket somewhere below?  
Was `clickHandler` empty?  Or is it a nested function with `createGrid` inside of it?

Reply:
yes. That's when I got frustrated and submitted. Thank you for all of your helpful comments.